src/client/css/partials/breaches.css

```diff
@@ -136,11 +136,19 @@
 
 .breaches-table header > :nth-child(3),
 .breaches-table summary > :nth-child(3) {
+  flex: 1 0 80px;
```

Comment:
Suggestion (non-blocking): Is there a `--padding-*` CSS variable in `variables.css` we could reuse or add for `flex-basis`?

Reply:
Agreed with this thought in general: UX is open to keeping in line with previously established vars.  

For this case though, I believe the number is _magic_: we need to choose something safe for different translation widths, but not so large it dominates the table.

The lack of a header title for this column also confuses/complicates it.  I explored using `display: grid` but it seemed more troublesome than flex.  I should have documented my pro/cons findings because I'm rethinking it now 😏

Reply:
Protocol does have a couple of "standard" magic values: https://protocol.mozilla.org/docs/fundamentals/design-tokens.html#spacing-units

We could use those, but they're still fairly arbitrary, so I don't have a strong opinion on whether we should. The existing columns don't, so for this PR I stayed consistent with that.